tests/RobotFramework/tests/cumulocity/telemetry/child_device_telemetry.robot

```diff
@@ -21,13 +21,13 @@ Child devices support sending custom measurements
 
 
 Child devices support sending custom events
-    Execute Command    tedge mqtt pub tedge/events/myCustomType1/${CHILD_SN} '{ "text": "Some test event", "someOtherCustomFragment": {"nested":{"value": "extra info"}} }'
+    Execute Command    tedge mqtt pub te/device/${CHILD_SN}///e/myCustomType1 '{ "text": "Some test event", "someOtherCustomFragment": {"nested":{"value": "extra info"}} }'
     ${events}=    Device Should Have Event/s    expected_text=Some test event    with_attachment=False    minimum=1    maximum=1    type=myCustomType1    fragment=someOtherCustomFragment
     Log    ${events}
 
 
 Child devices support sending custom events overriding the type
```

Comment:
This test gives false-positive to the current main branch. It cannot detect if two "type" fields are added as the test check only if the payload "contains" a type=otherType. 

In my opinion, this test can be covered by Rust code as below. https://github.com/thin-edge/thin-edge.io/blob/993a7c2d11fd1be501f1203ef07d434d60442967/crates/extensions/c8y_mapper_ext/src/converter.rs#L1480-L1496